src/components/career-form/career-file-upload-type.tsx

```diff
@@ -29,40 +40,159 @@ export const CareerFileUploadType = ({
-          File category
-        </option>
-        <option value="cv">CV</option>
-        <option value="cover-letter">Cover-Letter</option>
-        <option value="other">other</option>
-      </StyledSelect>
+        <ListboxButton arrow={<SelectArrow />} />
+        <ListboxPopover portal={false}>
+          <ListboxList>
+            <ListboxOption value={'default'} disabled>
+              File category
+            </ListboxOption>
+            <ListboxOption value="cv">CV</ListboxOption>
+            <ListboxOption value="cover-letter">Cover-Letter</ListboxOption>
+            <ListboxOption value="other">other</ListboxOption>
+          </ListboxList>
+        </ListboxPopover>
+      </StyledListboxInput>
       {errors.category_select?.[fileName[0]] && (
-        <FormError
-          error={{ message: 'Deine Auswahl fehlt' }}
-          lineHeight={100}
-        />
+        <FormErrorWrapper>
+          <FormError
+            error={{ message: 'Deine Auswahl fehlt' }}
+            lineHeight={90}
+          />
+        </FormErrorWrapper>
       )}
-    </div>
+    </SelectWrapper>
   );
 };
 
-const StyledSelect = styled.select`
-  border-radius: 4px;
+const SelectWrapper = styled.div`
+  display: block;
+  position: relative;
+
+  background: none;
+  border: none;
+  width: 100%;
+  margin-top: 10px;
+
+  ${up('md')} {
+    width: 144px;
+    margin-top: 0px;
+  }
+`;
+
+const FormErrorWrapper = styled.div`
+  margin-left: 16px;
+`;
+
+const StyledListboxInput = styled(ListboxInput)`
+  position: relative;
+  background: ${rgba('#ffffff', 0.1)};
+  border-radius: 20px;
+  border: none;
+  color: white;
+  font-weight: bold;
+  font-size: 14px;
+  cursor: pointer;  
+  width: calc(100% + 25px);
+  line-height: 110%;
+  
+  &:hover{
+    background: ${rgba('#ffffff', 0.2)};
+  }
+  
+  ${up('md')} {
+    width: 144px;   
+    height: 32px;
+  }
+  
+  [data-reach-listbox-button] {
+    border: none;
+    padding: 9px 16px 8px 16px;
+    cursor: pointer;
+    line-height: 1.15;
+    position: relative;
+    width: 100%;
+  }
+  
+  [data-reach-listbox-arrow] {
+    position: absolute;
+    right: 0px;
+    top: 13px;
+    right: 16px;
+    color: white;
+    width: 10px;
+  }
+  
+  [data-reach-listbox-popover] {
+     position: absolute;
+     top: 0px;
+     width: 100%;
+     background: ${rgba('#212940', 1)};
+     border: none;
+     border-radius: 10px;
+     padding: 0px;
+     z-index: 10;
+     
+     &:focus-within {
+      box-shadow: none;
+      outline: none;
+     }
+  }
+  
+  [data-reach-listbox-option] {  
+     padding: 8px 16px 8px 24px;
+     cursor: pointer;
+     line-height: 1.15;      
+     border-radius: 10px;
+     position: relative;
+    
+    &:hover{
+      background: ${rgba('#303B5C', 0.9)};
+    }
+    
+    &[aria-disabled="true"]:hover{
+      background: ${rgba('#212940', 1)};          
+    }
+    
+    &[data-current-selected]{
+        padding-left: 0px;  
+        &::before{
```

Comment:
double colon for pseudo elements and single colon for pseudo selectors, thank you for hearing my feedback 👍